### PR TITLE
Making snowcones edible

### PR DESCRIPTION
**Pull request by xXPawnStarrXx**
## About The Pull Request
Previous to now, snowcones could be made but not eaten. This allows people to actually eat them by giving them the icecream holder.
## Why It's Good For The Game
Cromchy ice good, food you can't eat bad.
## Changelog
:cl:
fix: snowcones can now successfully be placed inside your mouth.
/:cl:

### CODE REVIEW

File: code/game/objects/items/food/frozen.dm [hunk 1]
```diff
@@ -117,6 +121,10 @@
 	tastes = list("ice" = 1, "water" = 1, "limes" = 5)
 	foodtypes = FRUIT
 
+/obj/item/food/snowcones/lime/make_edible()
+	. = ..()
+	AddComponent(/datum/component/ice_cream_holder)
```
Comment (MrMelbert) [inline code/game/objects/items/food/frozen.dm:126]: Why are all of these necessary? Surely you only need to do this on the parent (`/obj/item/food/snowcones`)?

Comment (xXPawnStarrXx) [reply code/game/objects/items/food/frozen.dm:126]: I tried that, however it didn't seem to pass down in my testing.

Comment (MrMelbert) [reply code/game/objects/items/food/frozen.dm:126]: Well that doesn't make any sense, because I would wager all of these child procs are doing nothing

Comment (xXPawnStarrXx) [reply code/game/objects/items/food/frozen.dm:126]: I thought this was what fixed it, however it appears not. I now know what the actual problem is, I just have to work out why.

**Comment by SyncIt21** [comment]
Screenshot (367)
So like what is the point of this PR now?

**Comment by xXPawnStarrXx** [comment]
>So like what is the point of this PR now?

It was still drafted because I'm trying to work out the cause of the issue, I'm going to close it if I don't work it out by tonight, don't worry.

**Pull request closed without merging**